frontend/src/views/web/challenge/my-submission.html

```diff
@@ -1,163 +1,165 @@
-<section class="ev-sm-container ev-view ">
-    <div class="row">
-        <div class="col s12 m12">
-            <div class=" ev-card-panel ev-z-depth-5 ">
-                <div class="loader-container card-loader-container" id="loader" ng-show="challenge.isExistLoader" class="fade">
-                    <div class="loader">
-                        <div></div>
-                        <div></div>
-                        <div></div>
-                        <div></div>
-                        <div></div>
-                    </div>
+<section class="ev-sm-container ev-view challenge-container">
```

Comment:
@deshraj I think it would be good if we should reorder the fields under `my-submissions` Tab.
I think it should be -
`#, Team, Status, Submitted File, Result File, Stdout File, Stderr File, Method Name, Method Description, Project URL, Publication URL, Execution Time, Show on Leaderboard, Baseline,  Submitted At, `

The reason is since most participants don't fill the details for method name etc, showing these fields with None value is not good. Instead, we can use the space for which participants have to do the horizontal scroll every time such as result or stderr field. What do you think?

Reply:
Let's keep it as it is for now. I have a better idea in mind to use the collapsible (https://materializecss.com/collapsible.html). Then it will look nicer.

Reply:
yep, that makes sense.